api/events/schema.py

```diff
@@ -59,7 +50,7 @@ class Arguments:
     event = graphene.Field(Events)
 
     def mutate(self, info, **kwargs):
-        room_id = Events.check_status(self, info, **kwargs)
+        room_id = RoomSchedules().check_event_status(info, **kwargs)
         cancelled_event = EventsModel(
```

Comment:
I have a suggestion, I think it will be better if you get the room_id first before calling the `RoomSchedules().check_event_status` method.
I believe it is better that method should return a boolean (true or false).
Reason is `check_event_status` is returning room_id which is quite different from the name of the method.

Reply:
The calendar_id for the room comes from android when they hit this endpoint. So what I did when I want to populate events model the data doesn't come with the id of the room from the database, (the primary key of the room from the room model). So I used the calendar_id to make a query to the rooms model to get the room_id. And as far as the name of the method goes, I believe is accurate from what it does, it's responsible for giving back the status of the event incase of an error.

Reply:
And this is canceled event, there's not much Ive done here. test check-in instead @kampkelly

Reply:
Okay, understood. Please try and improve the test coverage.

Reply:
@kampkelly done, thanks for the feedback